### PR TITLE
Some infrastructure for dependent globular types

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: src/structured-types/dependent-reflexive-globular-types.lagda.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+[reflexive globular type](structured-types.reflexive-globular-types.md) `G`
+equipped with a reflexivity element `ρ`. A
+{{#concept "dependent reflexive globular type" Agda=Dependent-Reflexive-Globular-Type}}
+over `G` consists of
+[dependent globular type](structured-types.dependent-globular-types.md) `H` over
+`G` equipped with a reflexivity element `ρ'` consisting of
+
+```text
+ρ'₀ : (x : G₀) (y : H₀ x) → H₁ y y (ρ₀ x)
+ρ'₁ : {x x' : G₀} (y : H₀ x) (y' : H₀ x') → is-reflexive (H₁ y y')
```
Comment (fredrik-bakke) [inline src/structured-types/dependent-reflexive-globular-types.lagda.md:34]:
```suggestion
over `G` consists of a
[dependent globular type](structured-types.dependent-globular-types.md) `H` over
`G` equipped with a reflexivity element `ρ'` consisting of

```text
ρ'₀ : (x : G₀) (y : H₀ x) → H₁ y y (ρ₀ x)
ρ'₁ : {x x' : G₀} (y : H₀ x) (y' : H₀ x') → is-reflexive (H₁ y y').
```

**Review by fredrik-bakke — APPROVED** [review]
I had to look for a while to find something to correct. 😬 Otherwise, looks neat!

File: src/structured-types/globular-types.lagda.md [hunk 2]
```diff
@@ -206,3 +206,9 @@ globular-structure-Id A =
   .globular-structure-1-cell-globular-structure x y →
     globular-structure-Id (x ＝ y)
 ```
+
+## See also
+
+- [Reflexive globular types](structured-types.reflexive-globular-types.md)
+- [Symmetric globular types](structured-types.symmetric-globulary-types.md)
```
Comment (fredrik-bakke) [inline src/structured-types/globular-types.lagda.md:213]:
```suggestion
- [Symmetric globular types](structured-types.symmetric-globular-types.md)
```

**Comment by EgbertRijke** [comment]
Thanks so much for the swift review!

**Pull request merged**